### PR TITLE
Access point

### PR DESCRIPTION
**Pull request by Zainrax**
## API version update?
- Adds get on location to get current location

### CODE REVIEW

File: management-interface.go [hunk 1]
```diff
@@ -397,10 +436,13 @@ func addWPANetwork(ssid string, password string) error {
...
 		return errors.New("SSID cannot be bushnet")
 	}
+	if err := restartNetwork(); err != nil {
+		return err
+	}
 
 	networkExists, err := doesWPANetworkExist(ssid)
 	if err != nil {
-		return err
+		log.Println(err)
```
Comment (CameronRP) [inline management-interface.go:445]: Why is the error no longer returned here?

File: cmd/managementd/main.go [hunk 2]
```diff
@@ -114,6 +114,7 @@ func main() {
 	apiRouter.HandleFunc("/config", apiObj.SetConfig).Methods("POST")
 	apiRouter.HandleFunc("/clear-config-section", apiObj.ClearConfigSection).Methods("POST")
 	apiRouter.HandleFunc("/location", apiObj.SetLocation).Methods("POST") // Set location via a POST request.
+	apiRouter.HandleFunc("/location", apiObj.GetLocation).Methods("GET")  // Set location via a POST request.
```
Comment (CameronRP) [inline cmd/managementd/main.go:117]: Comment needs updated.

**Review by CameronRP — APPROVED** [review]
Looks good. Has been well tested :P Just a couple small things and you will have to merge the changes from main.

File: management-interface.go (hunk 1 above)
Comment (Zainrax) [reply management-interface.go:445]: This was due to wpa_cli not being available to used when in hotspot mode so it wasn't possible to add a new wifi network without it throwing an error. It still works in it's normal use cases

**Pull request merged**